tests/ui/consts/issue-83182.rs

```diff
@@ -1,9 +0,0 @@
-// Strip out raw byte dumps to make comparison platform-independent:
-// normalize-stderr-test "(the raw bytes of the constant) \(size: [0-9]*, align: [0-9]*\)" -> "$1 (size: $$SIZE, align: $$ALIGN)"
-// normalize-stderr-test "([0-9a-f][0-9a-f] |╾─*a(lloc)?[0-9]+(\+[a-z0-9]+)?─*╼ )+ *│.*" -> "HEX_DUMP"
-
-use std::mem;
-struct MyStr(str);
-const MYSTR_NO_INIT: &MyStr = unsafe { mem::transmute::<&[_], _>(&[&()]) };
```

Comment:
I deleted this test since this exact const is also tested in "raw-bytes", in a test case called `MYSTR_NO_INIT_ISSUE83182`.